### DISCUSSION TITLE
[Bug]: toggle mute resets the filter query in the explore page

### DISCUSSION THREAD

**Discussion by ivanjx** [blakeblackshear/frigate#15332] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

i usually look for events in the explore page because i can see each objects individually and also because it has more filters than the others. after i set up a filter, select an object, go to video tab and unmute the audio, the filter resets in the background and the scroll position goes back to top.

### Steps to reproduce

1. go to /explore
2. set some filter
![image](https://github.com/user-attachments/assets/724e7e86-ae15-45dc-bff9-0743b9f50e8d)
5. select an object
6. go to the video tab
![image](https://github.com/user-attachments/assets/b3e9af1f-7037-411f-a0a2-e08023f64855)
9. unmute the audio
![image](https://github.com/user-attachments/assets/a652857b-c1f0-4b63-8138-f21a418b9d76)
10. watch the browser url bar as the filter resets by itself
![image](https://github.com/user-attachments/assets/7f3b673b-b231-4447-8a8f-7e4c43147404)

### Version

latest dev

### In which browser(s) are you experiencing the issue with?

firefox 131

### Frigate config file

```yaml
irrelevant
```

### docker-compose file or Docker CLI command

```yaml
irrelevant
```

### Relevant Frigate log output

```shell
irrelevant
```

### Relevant go2rtc log output

```shell
irrelevant
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

tapo c325wb

### Screenshots of the Frigate UI's System metrics pages

irrelevant

**Comment by hawkeye217** [comment, marked as answer]
Thanks, fixed in https://github.com/blakeblackshear/frigate/pull/15334
